tests/toranj/cli/test-400-srp-client-server.py

```diff
@@ -61,11 +61,13 @@
 verify(server.srp_server_get_state() == 'disabled')
 verify(server.srp_server_get_addr_mode() == 'unicast')
 verify(client.srp_client_get_state() == 'Disabled')
-verify(client.srp_client_get_auto_start_mode() == 'Enabled')
 
 # Start server and client and register single service
 server.srp_server_enable()
 
+client.srp_client_enable_auto_start_mode()
+verify(client.srp_client_get_auto_start_mode() == 'Enabled')
+client.srp_client_enable_auto_start_mode()
```

Comment:
is this needed? it's already enabled in line 68

Reply:
Thanks. Removed it from the tests.